### ISSUE TITLE
Hide stdOut and stdErr output

### ISSUE THREAD

**Issue by VladUreche** [sbt-coursera/sbt-coursera#13]
One of the students added `println`-s to the methods and was able to see the incoming data, the methods called by the test suite and the output (and if the test was successful, that was exactly the reference output).

I would like to redirect `System.out` and `System.err` to `/dev/null` while the tests are being executed -- of course, that's easy to do with `setOut` and `setErr`, but since this may interact with loggers and the other process, I would rather discuss it first.

Ping @vjovanov @sjrd @heathermiller and @densh.

**Comment by vjovanov** [comment]
Makes sense - printing a lot could also crash the system (disk space, uploading to coursera, etc.)
